protocols/relay/src/copy_future.rs

```diff
@@ -73,7 +73,7 @@ where
         let this = &mut *self;
 
         loop {
-            if this.bytes_sent > this.max_circuit_bytes {
+            if this.max_circuit_bytes != 0 && this.bytes_sent > this.max_circuit_bytes {
```

Comment:
nit:
```suggestion
            if this.max_circuit_bytes > 0 && this.bytes_sent > this.max_circuit_bytes {
```
since `max_circuit_bytes` is an [`u64`](https://github.com/libp2p/rust-libp2p/blob/master/protocols/relay/src/behaviour.rs#L59)

Reply:
Since `max_circuit_bytes` is an unsigned integer, it can only be either 0 or non-zero positive integer right? In that case negetive integer can be ignored safely. But of course `>0` will make the code more robust. Will do!